plugins/formal-verification/testData/diagnostics/no_contracts/inheritance_fields.fir.diag.txt

```diff
@@ -0,0 +1,100 @@
+/inheritance_fields.kt:(208,215): info: Generated Viper text for createB:
+field class_A$member_fieldNotOverride: Ref
+
+field class_A$member_fieldOverride: Ref
+
+method class_B$constructor$fun_take$T_class_global$class_FieldB$return$T_class_global$class_B(local$fieldOverride: Ref)
+  returns (ret: Ref)
+  ensures dom$Type$isSubtype((dom$TypeOf$typeOf(ret): dom$Type), dom$Type$global$class_B())
+  ensures acc(ret.class_A$member_fieldNotOverride, wildcard)
+  ensures acc(ret.class_A$member_fieldOverride, wildcard)
+  ensures ret.class_A$member_fieldOverride == old(local$fieldOverride)
```

Comment:
Is the `old` here necessary?  I was under the impression that `old` is needed for heap values, but here you're using it on a local so I think it's a no-op.